models/caseload.js

```diff
@@ -0,0 +1,83 @@
+const BaseModel = require("./base/dynamo-db");
+const uuid = require("uuid");
+class CaseloadModel extends BaseModel {
```

Comment:
This class and CohortModel share a lot of the same code, should this extend from a different class other than base?

Reply:
Yes it technically should but it adds complications which I'd rather just leave for now. There's further changes to make in the base classes which would deprecate the extra code in these models.